packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx

```diff
@@ -44,24 +63,60 @@ const TeamPromptResponseCard = (props: Props) => {
   const responseStage = useFragment(
     graphql`
       fragment TeamPromptResponseCard_stage on TeamPromptResponseStage {
+        id
+        meetingId
+        meeting {
+          ... on TeamPromptMeeting {
+            localStageId
+            isRightDrawerOpen
+          }
+        }
         teamMember {
           userId
           picture
           preferredName
         }
+        discussion {
+          thread(first: 1000) @connection(key: "DiscussionThread_thread") {
+            edges {
+              ...TeamPromptRepliesAvatarList_edges
+            }
+          }
+        }
       }
     `,
     stageRef
   )
 
+  const onSelectDiscussion = () => {
+    if (meeting?.isRightDrawerOpen && meeting?.localStageId === responseStage.id) {
+      // If we're selecting a discussion that's already open, just close the drawer.
+      commitLocalUpdate(atmosphere, (store) => {
+        const meetingProxy = store.get(responseStage.meetingId)
+        if (!meetingProxy) return
+        meetingProxy.setValue(false, 'isRightDrawerOpen')
+      })
+    } else {
+      commitLocalUpdate(atmosphere, (store) => {
+        const meetingProxy = store.get(responseStage.meetingId)
+        if (!meetingProxy) return
+        meetingProxy.setValue(responseStage.id, 'localStageId')
+        meetingProxy.setValue(true, 'isRightDrawerOpen')
+      })
+    }
+  }
+
   const atmosphere = useAtmosphere()
   const {viewerId} = atmosphere
 
-  const {teamMember} = responseStage
+  const {teamMember, meeting, discussion} = responseStage
   const {picture, preferredName, userId} = teamMember
 
+  const discussionEdges = discussion.thread.edges
+  const replyCount = discussionEdges.length
+
   const isCurrentViewer = userId === viewerId
-  const isEmptyResponse = !isCurrentViewer // :TODO: (jmtaber129): Determine based on actual response, too
+  const isEmptyResponse = false // :TODO: (jmtaber129): Determine based on actual response, too
```

Comment:
-1 even though the upsert mutation isn't yet wired up we can improve here to use the real response or am I missing something?

Reply:
Same issue as in the drawer - this isn't implemented on the backend in this branch's base